run_dir/static/css/status.css

```
@@ -391,23 +392,21 @@ td.progress {
 .btn-imp{
   color: #fff;
   background-color: #a62100;
-  border-color: #571100;
  }
-.btn-imp:focus, .btn-imp:hover {
-  color: #fff;
-  background-color: #8a1b00;
-  border-color: #8a1b00;
-}
 .panel-important{
   background-color: #ffd9d9 !important;
 }
 .btn-devi{
   color: #fff;
   background-color: #a13a7e;
 }
-.btn-devi:focus, .btn-devi:hover {
+.btn-usr{
   color: #fff;
-  background-color: #a13a7e;
+  background-color: #ff6f00;
+}
+.btnCatFilter:focus, .btnCatFilter:hover {
```

* Add new CSS rule to give some left-margin to the button icons
* Update to use the new parent class name.

```suggestion
.rn-cat-filter button span {
    margin-left: 0.5rem;
}

.rn-cat-filter button:focus, .rn-cat-filter button:hover {
```